package.json

```diff
@@ -168,6 +168,11 @@
                 "type": "boolean",
                 "description": "Enable connection to running DAP Server",
                 "default": false
+              },
+              "dapodilDebugVersion": {
+                "type": "string",
+                "description": "Version of the dapoil debugger to run/use",
+                "default": "v0.0.5"
```

Comment:
We should avoid setting this by default so that the prompt for version is the default.

Setting the version is a convenience a user can add.